content/applications/inventory_and_mrp/purchase/advanced/analyze.rst

```diff
@@ -1,122 +1,146 @@
+click the :guilabel:`Measures` :icon:`fa-caret-down` button, and select one of the options from the
+drop-down menu:
+
+- :guilabel:`# of Lines`: The number of |PO| order lines, across all |POs|.
+- :guilabel:`Average Cost`: The average cost of |POs|.
+- :guilabel:`Days to Confirm`: The number of days it takes to confirm a |PO|.
+- :guilabel:`Days to Receive`: The number of days it takes to receive the products in a |PO|.
+- :guilabel:`Gross Weight`: The total weight of purchased products.
+- :guilabel:`Qty Billed`: The quantity of a product (or products) for which the vendor has already
+  been billed.
+- :guilabel:`Qty Ordered`: The quantity of a product (or products) ordered.
+- :guilabel:`Qty Received`: The quantity of an ordered product (or products) received.
+- :guilabel:`Qty to be Billed`: The quantity of an ordered product (or products) for which the
+  vendor has yet to be billed.
+- :guilabel:`Total`: The total amount spent, including tax.
+- :guilabel:`Untaxed Total`: The total amount spent, excluding tax. This measure is selected by
+  default.
+- :guilabel:`Volume`: The total volume of ordered products, for products which are measured by
+  volume.
+- :guilabel:`Count`: The total count of |POs|.
 
-If your company regularly purchases products, you have several reasons to track and analyze your
-performance. The insights gathered can help you to better forecast and plan your future orders. For
-example, you can assess if your business is dependent on particular vendors, and the data can help
-you negotiate discounts on prices.
-
-Generate customized reports
-===========================
-
-Go to *Reporting* to access the purchase analysis. Simply by accessing the reporting dashboard, you
-can get a quick overview of your actual performance. By default, the reporting dashboard displays a
-line chart presenting the untaxed amount of your purchase orders per day, and below it, key metrics
-and a pivot table.
-
-.. image:: analyze/analyze-reporting-dashboard.png
-   :align: center
-   :alt: Reporting dashboard in Odoo Purchase
-
-While the data initially presented is useful, there are several tools and features you can use to
-get even more insights on your purchases.
-
-Use filters to select the data you need
----------------------------------------
+.. tip::
+   Only one measure can be selected at a time when one of the :icon:`fa-area-chart`
+   :guilabel:`(graph view)` options is enabled. However, multiple measures, and varying group-by
+   criteria (on the x and y axes), can be selected when using the :icon:`oi-view-pivot`
+   :guilabel:`(pivot table)`.
 
-Odoo offers several default filters you can use and combine by clicking on *Filters*. When you
-select one or several filters, Odoo looks for all the orders that match at least one of the filters
-you have chosen and populates the chart, key metrics, and pivot table with the data. The
-pre-configured filters are:
+.. _purchase/advanced/analyze/example:
```

Comment:
In custom anchors, only the parent folder is optional to include, and with it being named "advanced", I think we can omit it, and just use a little more context :)
```suggestion
.. _purchase/purchase-analysis-example:
```